prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -55,6 +56,7 @@ abstract class Event(val eventAction: EventAction, val at: OffsetDateTime?) {
                 WIPE_ERROR -> TaskAction.wipe_error
                 RESEND -> TaskAction.resend
                 REBATCH -> TaskAction.rebatch
+                OTHER -> TaskAction.other
```

Comment:
Maybe just a comment here explaining that `OTHER` is not an expected value but rather a logical fallback default for `uploadBody`?

I could also see a tech debt ticket to investigate if `uploadBody` needs the default param at all

Reply:
I feel like in theory it shouldn't need the default, but.... there's defaults like this in a bunch of other places for some reason, which makes me feel like I might be missing something. Might be for graceful failure if something goes wonky with Azure/the queues.